civicapay_cashfile_sync_module/s3.tf

```diff
@@ -7,12 +7,12 @@ resource "aws_s3_bucket" "sftpbucket" {
 }
 
 # Create the /cashfiles folder in the S3 bucket
-resource "aws_s3_object" "s3_folder" {
-  depends_on   = [aws_s3_bucket.sftpbucket]
-  bucket       = aws_s3_bucket.sftpbucket.id
-  key          = "${var.bucket_folder_name}/"
-  content_type = local.s3_content_type
-}
+#resource "aws_s3_object" "s3_folder" {
+#  depends_on   = [aws_s3_bucket.sftpbucket]
+#  bucket       = aws_s3_bucket.sftpbucket.id
+#  key          = "${var.bucket_folder_name}/"
+#  content_type = local.s3_content_type
+#}
```

Comment:
Hmm... are we sure about removing it from TF?

I remember Adam mentioning something that the diff shown is not real when it comes to this bucket. Every time it says it wants to create it, but when you go to terraform apply, it never does create it.

It's possible that bucket did exist, but was simply not accounted for from within the TF state file maybe?

Reply:
Good point.  If the bucket does exist it will have to be added to the state at some point.  If it does exist, removing it from the terraform shouldn't destroy the bucket as it is not tracked, but it does remove it from the plan.  The databases have been destroyed so it can be returned.